draft-ietf-rats-corim.md

```diff
@@ -1669,9 +1669,9 @@ of the ACS `environment-map`.
 If there are no candidate entries then the triple containing the stateful environment does not match.
 
 A stateful environment `environment-map` is a subset of an ACS entry `environment-map`
-if each field which is present in the stateful environment `environment-map`
-(for example `class`, `instance` etc.)
-is also present in the ACS entry, and the CBOR encoded field values in the stateful environment and
+if each field (for example `class`, `instance` etc.) which is present in the
```

Comment:
```suggestion
if each field (for example `class-map`, `instance-id` etc.) which is present in the
```

Reply:
The fields in `environment-map` are named `class`, `instance` and `group` so I think using these names is correct.